PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditResponse.kt

```diff
@@ -0,0 +1,46 @@
+package com.braintreepayments.api.paypal.vaultedit
+
+import com.braintreepayments.api.BrowserSwitchOptions
+import com.braintreepayments.api.core.ExperimentalBetaApi
+
+@ExperimentalBetaApi
+sealed class PayPalVaultEditResponse {
+
+    /**
+     * The PayPal vault edit flow completed successfully.
+     *
+     * @property riskCorrelationId This ID is used to link subsequent retry attempts if payment is declined
+     */
+    class ReadyToLaunch internal constructor(
+        val riskCorrelationId: String,
+        val response: EditFIAgreementSetup,
+        internal var browserSwitchOptions: BrowserSwitchOptions? = null
```

Comment:
I don't think this should be optional/default to null since we need these options to launch the flow

Reply:
I think we have an [example here](https://github.com/braintree/braintree_android/blob/main/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalPaymentAuthRequestParams.kt) where the browserSwitchOptions are set to optional.